src/mixins/customerMixin.js

```diff
@@ -23,6 +24,28 @@ export default {
         },
       });
     },
+
+    updateMyCustomerPassword(currentPassword, newPassword) {
+      return this.$apollo.mutate({
+        mutation: gql`
+          mutation changePassword($version: Long!, $currentPassword: String!, $newPassword: String!) {
+            customerChangeMyPassword(version: $version, currentPassword: $currentPassword, newPassword: $newPassword) {
+              id
+              version
+              email
+            }
+          }`,
+        variables: {
+          version: this.me.customer?.version,
+          currentPassword,
+          newPassword,
+        },
+      })
+        .then((response) => {
```

Comment:
I would put the `then` in the Component (TabChangePassword) and not in the mixin. In the mixin I wouldn't put any post-actions, just the plain call to the backend.

Then you can use the `authMixin` that hides part of the complexity, as we should never call `auth.js` directly anywhere.